### PR TITLE
RH-DOCS-11626

### PR DESCRIPTION
**Pull request by Raghav2018**

### CODE REVIEW

**Comment by Raghav2018** [comment]
@shinyou21 @md3822 I have completed coding this sample app. Please review my code.

File: InsertExampleDataset/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+3. MongoDB instance. Either local or set up one for free on Mongo cloud [Atlas](https://www.mongodb.com/cloud/atlas).
+4. Download exampleDataset.json.
+
+## Check Your Environment
+1. Check for Java installation and Java driver dependency set up.
+
+## Consideration
+1. Get the correct connection string to your MongoDB instance.
+2. Get the correct file path on your local machine to ExampleDataset.json.
+
```
Comment (shinyou21) [inline InsertExampleDataset/README.md]: Check for libraries too

File: InsertExampleDataset/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+## Procedure
+1. Build the project.
+2. Use the following command to import the example dataset on to your MondoDB instance.
+```
```
Comment (shinyou21) [inline InsertExampleDataset/README.md]: Could you explain in detail how to build the project? the runnable jar file, run config etc

**Comment by Raghav2018** [comment]
@shinyou21  I have made the changes based on review comments. Let me know if it looks good.

**Review by shinyou21 — COMMENTED** [review]
LGTM

**Pull request merged**